packages/cdktf-cli/lib/cdktf-stack.ts

```diff
@@ -91,6 +91,20 @@ async function getTerraformClient(
       return tfClient;
     }
   }
+
+  if (parsedStack.terraform?.cloud) {
+    const tfClient = new TerraformCloud(
+      abortSignal,
+      stack,
+      parsedStack.terraform.cloud,
```

Comment:
`cloud` could include "tags" instead of "name" and would probably crash in that case. We should handle the case when users set tags (i.e. we could tell them they need to use the TF CLI in that case – if that's right).

Reply:
Yeah good point, I'll add a validation here